app/controllers/immunisation_imports/patients_controller.rb

```diff
@@ -30,4 +42,14 @@ def set_immunisation_import
   def set_patient
     @patient = @immunisation_import.patients.find(params[:id])
   end
+
+  def set_form
+    apply_changes =
+      params
+        .fetch(:patient_changes_form, {})
+        .permit(:apply_changes)
+        .fetch(:apply_changes, nil)
```

Comment:
Since you're only fetching one value, I think you can use a simpler form:

```suggestion
      params.dig(:patient_changes_form, :apply_changes)
```